android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/QuestionnaireExtension.kt

```diff
@@ -146,16 +146,18 @@ fun List<Questionnaire.QuestionnaireItemComponent>.prePopulateInitialValues(
   prePopulationParams: List<ActionParameter>
 ) {
   forEach { item ->
-    if (item.hasExtension(ITEM_INITIAL_EXPRESSION_URL)) {
-      item.removeExtension(ITEM_INITIAL_EXPRESSION_URL)
-    }
     prePopulationParams
       .firstOrNull {
         it.linkId == item.linkId &&
           !it.value.isNullOrEmpty() &&
           !it.value.contains(interpolationPrefix)
       }
       ?.let { actionParam ->
+
+        if (item.hasExtension(ITEM_INITIAL_EXPRESSION_URL)) {
```

Comment:
We don't want to ever remove the initial expressions

Reply:
I believe it was being removed in favour of the prepopulation configs. Ideally, we should not have both the prepolutaion config for a specific question and have the same question with an initialExpression since they both do the same thing. I might be wrong however